packages/node_modules/pouchdb-adapter-utils/src/parseDoc.js

```diff
@@ -90,7 +95,7 @@ function parseDoc(doc, newEdits) {
     if (!doc._id) {
       doc._id = uuid();
     }
-    newRevId = rev();
+    newRevId = rev(doc, dbOpts.useMd5Rev);
```

Comment:
A while back we switched from camel case to snake case for our options so for consistency should switch to that, I think `deterministic_rev` is best here, saying we use md5 is a leaky abstraction in case we need to stop using md5, `deteministic` is a bit long / unweidly however I doubt this is going to be use much, more of a precaution in case it causes unexpected problems when released